### PR TITLE
ZK-4631: Add option to choose toolbar overflow icon

### PR DESCRIPTION
**Pull request by b952426**
ZK-4631: Add option to choose toolbar overflow icon

### CODE REVIEW

File: zul/src/archive/web/js/zul/wgt/Toolbar.js [hunk 1]
```diff
@@ -70,7 +71,20 @@ zul.wgt.Toolbar = zk.$extends(zul.Widget, {
...
+		 * @since 9.6.0
+		 */
+		/**
+		 * Returns the iconSclass.
+		 * @return String
+		 * @since 9.6.0
+		 */
+		overflowPopupIconSclass: function (overflowPopupIconSclass) {
+			this.rerender();
+		},
```
Comment (b952426) [inline zul/src/archive/web/js/zul/wgt/Toolbar.js:87]: }

File: zul/src/archive/web/js/zul/wgt/mold/toolbar.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ function (out) {
 
 	if (this.isOverflowPopup()) {
 		out.push('<i id="', uuid, '-overflowpopup-button"',
-				' class="', this.$s('overflowpopup-button'), ' z-icon-ellipsis-h z-icon-fw" aria-hidden="true"></i>');
+				' class="', this.$s('overflowpopup-button'), ' ' + this.getOverflowPopupIconSclass(), ' z-icon-fw" aria-hidden="true"></i>');
```
Comment (b952426) [inline zul/src/archive/web/js/zul/wgt/mold/toolbar.js:32]: ,

File: zul/src/org/zkoss/zul/Toolbar.java [hunk 3]
```diff
@@ -146,5 +147,27 @@ protected void renderProperties(org.zkoss.zk.ui.sys.ContentRenderer renderer) th
 		if (!"start".equals(_align))
 			render(renderer, "align", _align);
 		render(renderer, "overflowPopup", _overflowPopup);
+		render(renderer, "overflowPopupIconSclass", getOverflowPopupIconSclass());
```
Comment (b952426) [inline zul/src/org/zkoss/zul/Toolbar.java:150]: if (!"start".equals(_align))
			render(renderer, "align", _align);

File: zul/src/archive/web/js/zul/wgt/Toolbar.js [hunk 4]
```diff
@@ -70,6 +71,21 @@ zul.wgt.Toolbar = zk.$extends(zul.Widget, {
...
+		 */
+		/**
+		 * Returns the iconSclass.
+		 * @return String
+		 * @since 9.6.0
+		 */
+		overflowPopupIconSclass: function (overflowPopupIconSclass) {
+			var icon = this.$n('overflowpopup-button');
+			if (this.desktop && icon)
+				icon.className = (this.$s('overflowpopup-button') + ' ' + overflowPopupIconSclass + ' z-icon-fw');
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wgt/Toolbar.js:88]: `()` no need

File: zul/src/archive/web/js/zul/wgt/mold/toolbar.js [hunk 5]
```diff
@@ -29,7 +29,7 @@ function (out) {
 
 	if (this.isOverflowPopup()) {
 		out.push('<i id="', uuid, '-overflowpopup-button"',
-				' class="', this.$s('overflowpopup-button'), ' z-icon-ellipsis-h z-icon-fw" aria-hidden="true"></i>');
+				' class="', this.$s('overflowpopup-button'), ' ', this.getOverflowPopupIconSclass(), ' z-icon-fw" aria-hidden="true"></i>');
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wgt/mold/toolbar.js:32]: _getIconXXX: function () {
   return this.$s('overflowpopup-button') + ' ' + overflowPopupIconSclass + ' z-icon-fw';
}

File: zul/src/org/zkoss/zul/Toolbar.java [hunk 6]
```diff
@@ -146,5 +147,28 @@ protected void renderProperties(org.zkoss.zk.ui.sys.ContentRenderer renderer) th
 		if (!"start".equals(_align))
 			render(renderer, "align", _align);
 		render(renderer, "overflowPopup", _overflowPopup);
+		if (!"z-icon-ellipsis-h".equals(_overflowPopupIconSclass))
+			render(renderer, "overflowPopupIconSclass", getOverflowPopupIconSclass());
+	}
+
+	/**
+	 * Returns the overflowPopupIconSclass name of this toolbar.
```
Comment (scribetw) [inline zul/src/org/zkoss/zul/Toolbar.java:155]: `@see overflowPopup`
Refine javadoc

**Review by scribetw — CHANGES_REQUESTED** [review]

File: zul/src/archive/web/js/zul/wgt/Toolbar.js [hunk 7]
```diff
@@ -70,6 +71,23 @@ zul.wgt.Toolbar = zk.$extends(zul.Widget, {
...
+		 * Sets the overflowPopupIconSclass.
+		 * When overflowPopup is true, toolbar has a button that shows a popup
+		 * users can customize the overflow popup icon.
+		 * Default: z-icon-ellipsis-h.
+		 * @since 9.6.0
+		 */
+		overflowPopupIconSclass: function () {
+			var icon = this.$n('overflowpopup-button');
+			if (this.desktop && icon)
+				icon.className = this._getOverflowpopupBtnClass();
```
Comment (scribetw) [inline zul/src/archive/web/js/zul/wgt/Toolbar.js:90]: OverflowPopup

**Review by scribetw — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Review by plijyfes — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**